utils/frame/frame-utilities-cli/src/pallet_id.rs

```diff
@@ -17,42 +17,42 @@
 
 //! Implementation of the `palletid` subcommand
 
+use clap::Parser;
 use frame_support::PalletId;
 use sc_cli::{
 	utils::print_from_uri, with_crypto_scheme, CryptoSchemeFlag, Error, KeystoreParams,
 	OutputTypeFlag,
 };
 use sp_core::crypto::{unwrap_or_default_ss58_version, Ss58AddressFormat, Ss58Codec};
 use sp_runtime::traits::AccountIdConversion;
-use structopt::StructOpt;
 
 /// The `palletid` command
-#[derive(Debug, StructOpt)]
-#[structopt(name = "palletid", about = "Inspect a module ID address")]
+#[derive(Debug, Parser)]
+#[clap(name = "palletid", about = "Inspect a module ID address")]
 pub struct PalletIdCmd {
 	/// The module ID used to derive the account
 	id: String,
 
 	/// network address format
-	#[structopt(
+	#[clap(
 		long,
 		value_name = "NETWORK",
 		possible_values = &Ss58AddressFormat::all_names()[..],
```

Comment:
No `arg_enum` here?

Reply:
`Ss58AddressFormat` (from `ss58-registry`):

```rust
/// A custom address format. See also [`Ss58AddressFormatRegistry`]
#[derive(Copy, Clone, PartialEq, Eq, Debug)]
pub struct Ss58AddressFormat {
	prefix: u16,
}
```

if we use `arg_enum` attribute here, `Ss58AddressFormat` should be like:

```rust
enum Ss58AddressFormat { ... }
```